core/src/main/java/org/mule/runtime/core/internal/policy/AbstractCompositePolicy.java

```diff
@@ -67,7 +67,7 @@ public AbstractCompositePolicy(List<Policy> policies,
    */
   public final Publisher<Event> processPolicies(Event operationEvent) {
     return just(operationEvent)
-        .flatMapMany(event -> processWithChildContext(event, new AbstractCompositePolicy.NextOperationCall()));
+        .flatMapMany(event -> processWithChildContext(event, new AbstractCompositePolicy.NextOperationCall(), null));
```

Comment:
There may be something wrong with that API. If there are use cases where the processWithChildContext method may be used outside of an element in the config, then it may not be a component location. In this case there's not since the AbstractCompositePolicy represents the chain of ALL the policies that applied to a source or an operation and that concept is not reflected in the XML.